xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3562,6 +3562,20 @@ bool CVideoPlayer::OpenVideoStream(CDVDStreamInfo& hint, bool reset)
   if (hint.stereo_mode.empty())
     hint.stereo_mode = CStereoscopicsManager::GetInstance().DetectStereoModeByString(m_item.GetPath());
 
+  {
+    SelectionStream& s = m_SelectionStreams.Get(STREAM_VIDEO, 0);
+    switch(CMediaSettings::GetInstance().GetCurrentVideoSettings().m_StereoMode)
+    {
+      case RENDER_STEREO_MODE_SPLIT_VERTICAL:   s.stereo_mode = "left_right"; break;
```

Comment:
please use a different coding style

1) vertical alignment sooner or later breaks because there is always some variable to add that is longer than the ones before
2) only a single statement per line or setting break points is a nightmare